bin/entry.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env sh
 
-bundle exec rails db:create
-bundle exec rails db:migrate
+bundle exec rake db:create
+bundle exec rake db:migrate
```

Comment:
There is also `db:setup` that does both (and a bit more).

Reply:
whats that "a bit more"?